conans/client/conf/__init__.py

```diff
@@ -80,8 +80,10 @@
         version: ["3.3", "3.4", "3.5", "3.6", "3.7", "3.8", "3.9", "4.0",
                   "5.0", "6.0", "7.0", "7.1",
                   "8"]
-        libcxx: [libstdc++, libstdc++11, libc++, c++_shared, c++_static]
+        libcxx: [None, libstdc++, libstdc++11, libc++, c++_shared, c++_static]
         cppstd: [None, 98, gnu98, 11, gnu11, 14, gnu14, 17, gnu17, 20, gnu20]
+        driver: [None, cl, g++]
```

Comment:
I don't think the driver belongs here.

Reply:
if it doesn't belong here, probably it doesn't belong to settings at all, right?
in that case, I'll just remove it